src/main/groovy/edu/oregonstate/mist/osuevents/db/EventsDAO.groovy

```diff
@@ -44,8 +44,9 @@ public interface EventsDAO extends Closeable {
             OWNER
         FROM EVENTS_EVENTS
         WHERE (EVENT_ID = :eventID OR :eventID IS NULL)
-	 AND (CREATED_AT >= SYSDATE - :changedInPastHours/24 OR :changedInPastHours IS NULL)
-	 AND (UPDATED_AT >= SYSDATE - :changedInPastHours/24 OR :changedInPastHours IS NULL)
+        AND (CREATED_AT >= SYSDATE - :changedInPastHours/24
+            OR UPDATED_AT >= SYSDATE - :changedInPastHours/24
+            OR :changedInPastHours IS NULL)
```

Comment:
Yes, these conditions should definitely be OR and not AND. I tested with the changes and it worked as expected (events that had been created recently but not updated were returned).

Reply:
Thanks for your help testing.